aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -113,6 +113,9 @@ def define(cls, spec):
             message='The initialization calculation failed.')
         spec.exit_code(501, 'ERROR_IONIC_CONVERGENCE_REACHED_EXCEPT_IN_FINAL_SCF',
             message='Then ionic minimization cycle converged but the thresholds are exceeded in the final SCF.')
+        spec.exit_code(502, 'WARNING_ELECTRONIC_CONVERGENCE_NOT_REACHED',
```

Comment:
```suggestion
        spec.exit_code(710, 'WARNING_ELECTRONIC_CONVERGENCE_NOT_REACHED',
```

Reply:
Wondering if we should maybe try and maintain the same exit codes as much as possible. Not sure if this will be possible but I can imagine that it might be confusing if exit codes keep switching